### ISSUE TITLE
Bug 54108845: [React Native Gallery -> ProgressView]: Screen reader is announcing incorrect progress value for the “Indeterminate” progress bar.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#510]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘ProgressView’ button present under “Basic Input” group and invoke it.
Navigate to “Indeterminate” progress bar present under “An indeterminate ProgressView” heading using caps + arrow keys.
Listen to narrator announcement.
Actual Result:
Screen reader is announcing incorrect progress value for the “Indeterminate” progress bar. When user navigates to it using caps + arrow keys narrator is announcing as “0% progress bar”.
 
Expected Result:
Screen reader should announce proper progress value for the “Indeterminate” progress bar. When user navigates to it narrator should announce as “Indeterminate progress bar”.
 
User Impact:
Users with visual impairments who rely on screen reader will be mis guided if narrator announces incorrect progress value information to the user.
 
Link to ADO bug: https://microsoft.visualstudio.com/OS/_workitems/edit/54108845

**Comment by Yajur-Grover** [comment]
This should be able to be resolved by changing the `accessibilityLabel` value, so that the screen reader announces the correct output. Feel free to contact me for any questions.
